syft/core/hooks/torch/hook.py

```diff
@@ -275,7 +157,9 @@ def _get_overload_method_in_tensor_or_var(hook_self, method, isfunc=False):
         accordingly.
         """
         @functools.wraps(method)
+        # def method_router(self, *args, **kwargs):
         def method_router(self, *args, **kwargs):
+
```

Comment:
Unless there's a really persuasive reason not to, I think we should stick with [PEP 257](https://www.python.org/dev/peps/pep-0257/) for formatting docstrings.

Reply:
Forgive me - I'm not 100% sure which part of this violated PEP 257 - reading the link you sent trying to find it though.

Reply:
was it the newline?

Reply:
cool - so does that look right now?